docs/fsharp/tutorials/getting-started/getting-started-vscode.md

```diff
@@ -1,4 +1,278 @@
+
+4. Install the [Visual F# Tools](https://www.microsoft.com/en-us/download/details.aspx?id=48179).
+
+On 64-bit Windows, the compiler and tools are located here:
+
+```
+C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsc.exe
+C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsi.exe
+C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsiAnyCpu.exe
+```
+
+On 32-bit Windows, the compiler tools are located here:
+
+```
+C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsc.exe
+C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsi.exe
+C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsiAnyCpu.exe
+```
+
+Ionide automatically detects the compiler and tools, but if it doesn't for some reason, you can manually add the containing folder (`...\Microsoft SDKs\F#\4.0`) to your PATH.
+
+### macOS
+
+On macOS, Ionide uses Mono.  The easiest way to install Mono on macOS is via Homebrew.  Simply type the following into your terminal:
+
+```
+brew install mono
+```
+
+### Linux
+
+On Linux, Ionide also uses Mono.  If you're on Debian or Ubuntu, you can use the following:
+
+```
+sudo apt-get update
+sudo apt-get install mono-complete fsharp
+```
+
+## Installing Visual Studio Code and the Ionide plugin
+
+You can install Visual Studio Code on the [code.visualstudio.com](https://code.visualstudio.com) website.  After that, there are two ways to find the Ionide plugin:
+
+1. Use the Command Palette (Ctrl-P on Windows, ⌘-P on macOS, Ctrl-Shift-P on Linux) and type the following:
+
+    ```
+    >ext install Ionide
+    ```
+
+2. Click the Extensions icon and search for "Ionide":
+
+    ![](media/getting-started-vscode/vscode-ext.png)
+
+The only plugin required is **Ionide-fsharp**.  However, you can also install **Ionide-FAKE** and **Ionide-Paket** to get [FAKE](http://fsharp.github.io/FAKE/) and [Paket](https://fsprojects.github.io/Paket/) support.
+
+## Creating your first project Ionide
+
+To create a new F# project, open the Command Palette (ctrl-P on Windows, ⌘-P on macOS, ctrl-shift-P on Linux) and type the following:
+
+```
+>f#: New Project
+```
+
+This is powered by the [FORGE](https://github.com/fsprojects/Forge) project.  You should see something like this:
+
+![](media/getting-started-vscode/vscode-new-proj.png)
+
+Select "F#: New Project" by hitting **Enter**, which will take you to this step:
+
+![](media/getting-started-vscode/vscode-proj-type.png)
+
+This will select a template for a specific type of project.  There are quite a few options here, such as an [FsLab](http://fslab.org) template for Data Science or [Suave](https://suave.io) template for Web Programming.  This article uses the `classlib` template, so highlight that and hit **Enter**.  You will then reach this step:
```

Comment:
should add a note that if you don't see any templates listed you need to run the refresh templates command